core/rawdb/chain_freezer.go

```diff
@@ -413,6 +429,21 @@ func (f *chainFreezer) SetupFreezerEnv(env *ethdb.FreezerEnv) error {
 	return nil
 }
 
+func (f *chainFreezer) checkFreezerEnv() error {
+	_, exist := f.freezeEnv.Load().(*ethdb.FreezerEnv)
+	if exist {
+		return nil
+	}
+	blobFrozen, err := f.TableAncients(ChainFreezerBlobSidecarTable)
+	if err != nil {
+		return err
+	}
+	if blobFrozen > 0 {
```

Comment:
How does this condition means freezer env is missing? A blob can still freeze by default right?

Reply:
yes, this condition means that env must be guaranteed to exist when the blob table already has data.